newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -0,0 +1,170 @@
+// @flow
+
+import * as React from 'react';
+import { t, Trans } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import { type I18n as I18nType } from '@lingui/core';
+
+import { Card } from '@material-ui/core';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import CheckCircleIcon from '@material-ui/icons//CheckCircle';
+
+import { ResponsiveLineStackLayout } from '../../UI/Layout';
+import CardContent from '../../UI/Card/CardContent';
+import Text from '../../UI/Text';
+import { Column, LargeSpacer, Line, Spacer } from '../../UI/Grid';
+import IconButton from '../../UI/IconButton';
+import GDevelopThemeContext from '../../UI/Theme/ThemeContext';
+
+import Rating from './Rating';
+
+import {
+  updateComment,
+  type Comment,
+  type GameRatings,
+} from '../../Utils/GDevelopServices/Play';
+import { useDebounce } from '../../Utils/UseDebounce';
+import { type AuthenticatedUser } from '../../Profile/AuthenticatedUserContext';
+import { showErrorBox } from '../../UI/Messages/MessageBox';
+
+const styles = { textComment: { whiteSpace: 'pre-wrap' } };
+
+type Props = {|
+  comment: Comment,
+  authenticatedUser: AuthenticatedUser,
+|};
+
+const getRatings = (ratings: ?GameRatings) => {
+  if (!ratings) return null;
+  if (ratings.version === 1) {
+    return [
+      {
+        key: 'rating-sound',
+        label: <Trans>Sound</Trans>,
+        value: ratings.sound,
+      },
+      {
+        key: 'rating-visuals',
+        label: <Trans>Visuals</Trans>,
+        value: ratings.visuals,
+      },
+      { key: 'rating-fun', label: <Trans>Fun</Trans>, value: ratings.fun },
+      {
+        key: 'rating-ease-of-use',
+        label: <Trans>Ease of use</Trans>,
+        value: ratings.easeOfUse,
+      },
+    ];
+  }
+};
+
+const getCurrentState = (pendingState: ?boolean, state: boolean) => {
+  return pendingState === null ? state : pendingState;
+};
+
+const FeedbackCard = ({ comment, authenticatedUser }: Props) => {
+  const { getAuthorizationHeader, profile } = authenticatedUser;
+  const ratings = getRatings(comment.ratings);
+  const theme = React.useContext(GDevelopThemeContext);
+
+  const [processed, setProcessed] = React.useState<boolean>(
```

Comment:
Not mandatory for this PR, but think of if a hook "useOptimisticState" could be done to abstract this. You would pass it the initial state (source of truth), the function to do the change, and it would return the state to display (i.e: the pending or the source of truth) and the function to call to change it (that would change the pending state, and would launch the debounced version of the function to do the change).